identity/webapp/src/frontend/Registration/OldRegistration.tsx

```diff
@@ -74,7 +74,7 @@ export const OldRegistration: React.FC = () => {
           .then(() => {
             setCreated(true);
           })
-          .catch((error) => {
+          .catch(error => {
```

Comment:
Something wrong with Digirati's Prettier setup, maybe?

Reply:
Not sure, we'll need to dig into this.